src/biome/text/api_new/configuration.py

```diff
@@ -213,14 +213,16 @@ def __init__(
 
 
 class VocabularyConfiguration:
-    """Configures a `Vocabulary` before it gets created from data
+    """Configures a ``Vocabulary`` before it gets created from data
 
-    Use this to configure a Vocabulary using specific arguments from `allennlp.data.Vocabulary`
+    Use this to configure a Vocabulary using specific arguments from `allennlp.data.Vocabulary``
 
     See [AllenNLP Vocabulary docs](https://docs.allennlp.org/master/api/data/vocabulary/#vocabulary])
 
     Parameters
     ----------
+    from_path: ``Optional[str]``
+        If provided, try to load model vocab from specified folder path
```

Comment:
If provided, try to load model vocab from specified folder path containing datasource YAML definitions ?

Reply:
No, the folder will contains an already saved vocab

Reply:
It will use the path for load a vocab using `Vocabulary.from_files` method